starboard/user.h

```diff
@@ -55,6 +56,7 @@ typedef enum SbUserPropertyId {
   // A unique user ID of a user.
   kSbUserPropertyUserId,
 } SbUserPropertyId;
+#endif
 
 // Well-defined value for an invalid user.
 #define kSbUserInvalid (SbUser) NULL
```

Comment:
To be clear, we expect this entire file to gradually get fully #ifdefd out later, right ?